db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/VMRetirement.class/__methods__/update_retirement_status.rb

```diff
@@ -1,31 +1,45 @@
 #
-# Description: This method updates retirement status
+# Description: This method updates the vm retirement status.
+# Required inputs: status
 #
 
 # Get variables from Server object
 server = $evm.root['miq_server']
 
 # Get State Machine
-state  = $evm.current_object.class_name
-status = $evm.inputs['status']
+state = $evm.current_object.class_name
 
 # Get current step
 step = $evm.current_object.current_field_name
 
+# Get status from input field status
+status = $evm.inputs['status']
+
 # Get status_state ['on_entry', 'on_exit', 'on_error']
 status_state = $evm.root['ae_status_state']
 
-$evm.log("info", "Server:<#{server.name}> Ae_Result:<#{$evm.root['ae_result']}> State:<#{state}>")
-$evm.log("info", "Step:<#{step}> Status_State:<#{status_state}> Status:<#{status}>")
-
 vm = $evm.root['vm']
 
+$evm.log("info", "Server:<#{server.name}> Ae_Result:<#{$evm.root['ae_result']}> State:<#{state}> Step:<#{step}>")
+$evm.log("info", "Status_State:<#{status_state}> Status:<#{status}>")
+
+# Update Status Message
+updated_message  = "Server [#{server.name}] "
+updated_message += "VM [#{vm.name}] " if vm
+updated_message += "Step [#{step}] "
```

Comment:
@mkanoor @billfitzgerald0120 We can modify this code when we start using the description field as we discussed previously.